src/apscheduler/_schedulers/async_.py

```diff
@@ -529,6 +532,64 @@ async def remove_schedule(self, id: str) -> None:
         self._check_initialized()
         await self.data_store.remove_schedules({id})
 
+    async def pause_schedule(self, schedule_id: str) -> None:
+        """Pause the specified schedule."""
+        self._check_initialized()
+        await self.data_store.add_schedule(
+            schedule=attrs.evolve(await self.get_schedule(schedule_id), paused=True),
+            conflict_policy=ConflictPolicy.replace,
+        )
+
+    def _get_unpaused_next_fire_time(
```

Comment:
This method doesn't seem to be used anywhere but `unpause_schedule()`. Therefore it should be inlined, if there's no justification for it to be a separate method.